app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -64,6 +64,20 @@ def parse_compute_resource(object, kind, props)
   end
   alias parse_cluster_compute_resource parse_compute_resource
 
+  def parse_customization_spec_manager(_object, kind, props)
+    return if kind == "leave"
+
+    props[:info].to_a.each do |spec_info|
+      persister.customization_specs.build(
+        :name             => spec_info[:name],
+        :typ              => spec_info[:type],
```

Comment:
I think it should be :type

Reply:
`:type` is a reserved column name used by STI, `:typ` is correct here used to not conflict with STI.